bin/main.rb

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env ruby
+require_relative '../lib/board.rb'
+require_relative '../lib/player.rb'
+require_relative '../lib/game.rb'
+
+
+board = Board.new
+player1 = Player.new
+player2 = Player.new
+ui = UI.new
+
+answer = Game.new(board, player1, player2, ui)
+until answer.continue == "N"
```

Comment:
We probably want this logic to reside in a module and just call it here. Essentially what we want for our main executable file is just setting some **necessary** global variables and calling the **necessary** methods:

```
board = Board.new
player1 = Player.new
player2 = Player.new
ui = UI.new
answer = Game.new(board, player1, player2, ui)

this_method_used_to_be_a_full_logic(answer)
```